src/css/profile/mobile/common/maintab.less

```diff
@@ -11,6 +11,7 @@
 	background-color: var(--bottom-bar-color);
 	position: fixed;
 	bottom: 0;
+	z-index: 1000;
```

Comment:
Is there any reason to set the value to 1000?
Wouldn't the other cases overlap?

Reply:
I'll remove it.